frontend/src/pages/Assessments/Assessments.tsx

```diff
@@ -56,11 +65,23 @@ const Assessments = () => {
         <LandingPageText>
           assessments
           <Prompt>Select the correct images to match the target words.</Prompt>
-          <PurpleButton text={"begin"} top={100} />
+          <PurpleButton
+            text={"begin"}
+            top={100}
+            onClick={() => {
+              // hardcoding assessment with id=1 for now
+              history.push("/assessments/1");
+              getWords();
```

Comment:
Seems weird that we call this and don't actually use the assessment. I am not sure what the correct answer is lol, but this seems kind of weird to me.